node/endpoint-handler.js

```diff
@@ -84,12 +84,12 @@ TChannelEndpointHandler.prototype.handleArg1 = function handleArg1(req, buildRes
             'no such endpoint service=%j endpoint=%j',
             req.service, name));
     } else if (handler.canStream) {
-        handler(req, buildResponse);
+        handler.call(self, req, buildResponse);
```

Comment:
I’m going to push to uniformly accept handleRequest functions or handler.handleRequest objects in all these places that take handler. This has implications for how that would work. Might be better to pass the endpoint handler as an argument.

See unmerged: https://github.com/uber/tchannel/blob/circuit-breaker/node/request-handler.js